### PR TITLE
Upgrade to GitLab CE 15.5.0

### PR DESCRIPTION
**Pull request by antt1995**

### CODE REVIEW

**Comment by sachilles** [comment]
@antt1995 Wow, that's fast. Still, I think we should wait for the official release. (Tomorrow will be the day).

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+FROM ubuntu:focal-20220922
 
-ARG VERSION=15.4.2
+ARG VERSION=15.5.0
 
 ENV GITLAB_VERSION=${VERSION} \
     RUBY_VERSION=2.7.6 \
     RUBY_SOURCE_SHA256SUM="e7203b0cc09442ed2c08936d483f8ac140ec1c72e37bb5c401646b7866cb5d10" \
     GOLANG_VERSION=1.17.13 \
     GITLAB_SHELL_VERSION=14.10.0 \
```
Comment (sachilles) [inline Dockerfile:9]: Please upgrade to 14.12.0.

File: Changelog.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
 
 This file only reflects the changes that are made in this image. Please refer to the upstream GitLab [CHANGELOG](https://gitlab.com/gitlab-org/gitlab-foss/blob/master/CHANGELOG.md) for the list of changes in GitLab.
 
+**15.5.0**
+
+- gitlab: upgrade CE to v15.5.0
+- gitaly: upgrade to v15.5.0
+
```
Comment (sachilles) [inline Changelog.md:9]: Please add 

- gitlab-shell: upgrade to v14.12.0

**Review by sachilles — CHANGES_REQUESTED** [review]
Please update the gitlab-shell packages.

**Comment by antt1995** [comment]
@sachilles All done as requested

**Review by sachilles — APPROVED** [review]
Many thanks!

**Pull request merged**